### ISSUE TITLE
Stop server from starting if update in progress

### ISSUE THREAD

**Issue by Log1x** [FezVrasta/ark-server-tools#566]
Hi.

I currently run 4 instances of arkmanager on my dedicated box and have had a great experience so far!

One thing I found to be problematic though is when doing `arkmanager start @all` with auto-update on start enabled along with a cronjob to automatically check for updates, if you do it during an update the start script will tell you that an update is already in progress and then start the server anyway while it's being updated.

This can lead to issues and potential corruption. Anyway to store the steamcmd.sh update pid in a file or something per-instance and then make the server not start if its currently being updated (or start after it's finished)?

My cronjob's are as followed:

```
*/15 * * * * /usr/local/bin/arkmanager update @pve --warn --saveworld --update-mods --backup >/dev/null 2>&1
*/15 * * * * /usr/local/bin/arkmanager update @pvp --warn --saveworld --update-mods --backup >/dev/null 2>&1
*/15 * * * * /usr/local/bin/arkmanager update @valhalla --warn --saveworld --update-mods --backup >/dev/null 2>&1  
*/15 * * * * /usr/local/bin/arkmanager update @test --warn --saveworld --update-mods --backup >/dev/null 2>&1
```

along with:

```
arkAutoUpdateOnStart="true"
```

Here's an example of this happening (and ending in a server that is not joinable due to mod corruption and client side compatibility issues):

```
steam@dev:~/servers/arkonia_server$ arkmanager start @pve
Running command 'start' for instance 'pve'
[  ERROR  ]     Mod 576432474 is requested but not installed.  Run 'arkmanager installmod 576432474' to install this mod.
Updating server
Update already in progress (PID: 34621)
The server is starting...
The server is now running, and should be up within 10 minutes
```

Fortunately with how awesome the arkmanager backup scripts are (and also having them as a cronjob) this didn't result in any major issues,  but I still feel a fix should definitely be looked into to prevent this from happening whether it be a cronjob or human error (accidentally starting twice while having 2 terminals open, etc.)

Thank you!

**Comment by klightspeed** [comment]
I have pushed a fix for this (653006a) to master.

**Comment by Log1x** [comment]
Can we get this fix in 1.6-dev as well?

Thanks

**Comment by klightspeed** [comment]
It is now in 1.6-dev and 1.6-dev.unstable
